.circleci/continue_config.yml

```diff
@@ -2941,6 +2944,83 @@ jobs:
             sed -i -e "s/crate-type.*$/crate-type = [\"rlib\"]/g" Cargo.toml
             cargo clippy --all-targets --all-features -- -D warnings -Aunknown-lints
 
+  "compile_loader":
+    working_directory: ~/datadog
+    parameters:
+      dd_library_package_url:
+        type: string
+      docker_image:
+        type: string
+      resource_class:
+        type: string
+        default: medium
+    <<: *BARE_DOCKER_MACHINE
+    environment:
+    steps:
+      - restore_cache:
+          keys:
+            - source-v1-{{ .Branch }}-{{ .Revision }}
+      - <<: *STEP_CHECKOUT
+      - <<: *STEP_ATTACH_WORKSPACE
+      - setup_docker:
+          docker_image: << parameters.docker_image >>
+      - run:
+          name: Compile the loader extension
+          # FIXME:  -Wextra -Werror
+          command: |
+            set -eo pipefail
+            cd loader
+            phpize && ./configure
+            make clean && make -j all ECHO_ARG="-e" CFLAGS="-std=gnu11 -O2 -g -Wall"
+      - run:
+          name: Download the dd-trace-php package
+          command: |
+            set -eo pipefail
+            cd loader
+            curl -Lo dd-library-php.tar.gz << parameters.dd_library_package_url >>
+            tar xzf dd-library-php.tar.gz
```

Comment:
I suppose that's temporary, and we're going to put this pipeline after _package extension_?
Like, having the test_loader job depend on compile_loader and package extension.

Reply:
Yes. 
I did this to debug the CI pipeline faster